Formula/vasm.rb

```diff
@@ -0,0 +1,84 @@
+class Vasm < Formula
+  desc "Portable and retargetable assembler"
+  homepage "http://sun.hasenbraten.de/vasm/"
+  url "http://server.owl.de/~frank/tags/vasm1_7e.tar.gz"
+  version "1.7e"
+  sha256 "2878c9c62bd7b33379111a66649f6de7f9267568946c097ffb7c08f0acd0df92"
+
+  option "with-6502", "Enable 6502 CPU target (binary vasm6502_SYNTAX)"
```

Comment:
Just build the default set and maybe add one other option `with-all` to build the rest too.